src/modules/flyout/flyout.service.spec.ts

```diff
@@ -1,138 +1,133 @@
-        BrowserModule,
-        SkyFlyoutModule,
-        SkyFlyoutFixturesModule,
-        BrowserAnimationsModule
-      ]
-    });
-  });
-
-  beforeEach(
-    inject(
-      [
-        SkyFlyoutService,
-        ApplicationRef
+        NoopAnimationsModule
       ],
-      (
-        _flyoutService: SkyFlyoutService,
-        _applicationRef: ApplicationRef
-      ) => {
-        flyoutService = _flyoutService;
-        flyoutService.close();
-        applicationRef = _applicationRef;
-      }
-    )
-  );
-
-  it('should show a flyout and return an instance that can then be closed',
-  fakeAsync(() => {
-    openFlyout(FlyoutTestComponent);
-    applicationRef.tick();
-
-    expect(document.body.querySelector('.sky-flyout')).toExist();
-    closeFlyout();
-    tick();
-    applicationRef.tick();
-
-    expect(document.body.querySelector('.sky-flyout')).not.toExist();
-  }));
-
-  it('should not show multiple flyouts', fakeAsync(() => {
-    openFlyout(FlyoutTestComponent);
-    openFlyout(FlyoutTestComponent);
-    tick();
-
-    expect(document.body.querySelector('.sky-flyout')).toExist();
-    expect(document.body.querySelectorAll('.sky-flyout').length).toBe(1);
-    tick();
-    applicationRef.tick();
-
-    expect(document.body.querySelector('.sky-flyout')).toExist();
-    expect(document.body.querySelectorAll('.sky-flyout').length).toBe(1);
-    tick();
-    applicationRef.tick();
-
-    closeFlyout();
-    tick();
-    applicationRef.tick();
-
-    expect(document.body.querySelector('.sky-flyout')).not.toExist();
-  }));
-
-  it('should allow data to be passed to the modal component when opened', fakeAsync(() => {
-    const flyoutInstance = openFlyout(FlyoutWithValuesTestComponent, {providers: [
-      {
-        provide: FlyoutTestValues,
-        useValue: {
-          valueA: 'A'
-        }
-      }
-    ]});
-
-    expect(flyoutInstance.componentInstance.values.valueA).toBe('A');
-
-    closeFlyout();
-  }));
-
-  it('should close the host comopnent when the host component sends the closed event', (done) => {
-    const flyoutInstance = openFlyout(FlyoutTestComponent);
-    applicationRef.tick();
-
-    expect(document.body.querySelector('.sky-flyout')).toExist();
-    expect(document.body.querySelectorAll('.sky-flyout').length).toBe(1);
-
-    flyoutInstance.closed.subscribe(() => {
-      setTimeout(() => {
-        expect(document.body.querySelector('.sky-flyout')).not.toExist();
-        done();
-      }, 10);
+      providers: [
+        SkyFlyoutService,
+        {
+          provide: SkyFlyoutAdapterService,
+          useValue: {
+            appendToBody() { },
+            removeHostElement() { }
+          }
+        },
+        {
+          provide: ApplicationRef,
+          useValue: {
+            attachView() {},
+            detachView() {}
+          }
+        },
+        Injector,
+        {
+          provide: ComponentFactoryResolver,
+          useValue: {
+            resolveComponentFactory() {
```

Comment:
Woah..... lol

Reply:
HAHA, yeah, I was debating killing this spec to begin with...